src/main/jbake/content/hawkular-apm/index.adoc

```diff
@@ -0,0 +1,124 @@
+= Hawkular APM
+Gary Brown
+2016-11-07
+:description: OpenTracing compliant application performance management
+:jbake-type: page
+:jbake-status: published
+:icons: font
+:toc: macro
+:toc-title:
+
+toc::[]
+
+
+== What is Application Performance Management?
+
+As defined on https://en.wikipedia.org/wiki/Application_performance_management[wikipedia], _"application performance management (APM) is the monitoring and management of performance and availability of software applications. APM strives to detect and diagnose complex application performance problems to maintain an expected level of service"_.
+
+In a microservices world this becomes even more critical due to the potentially large number of simple services that collaborate to deliver a business application.
+
+Being able to obtain high level aggregated views of an application's performance, as well as trace the individual path taken by a particular end user request through a potentially dynamically scaled cloud environment, enables application operations and development teams to understand where issues may have occurred and what improvements can be made to deliver better performance for the business.
+
+
+== Features
+
+This section provides a high level description of the features available in Hawkular APM. For more detailed information, including how install and use the project, please see the https://hawkular.gitbooks.io/hawkular-apm-user-guide/content/[user guide].
```

Comment:
`user guide` or `documentation` ? In top menu this page is denoted as documentation.